docs/gettingstarted/query_test_report.rst

```diff
@@ -20,6 +20,19 @@ with default format fields. To see a list of all format fields, click :ref:`here
     .. command-output:: buildtest report
        :ellipsis: 20
 
+The ``buildtest report`` command will fetch all records and display them in a table format, if you want to know how many records are displayed in the table you can use ``buildtest report --row-count`` which will display   
+a raw count of records in the table. In the example below we will run ``buildtest report --row-count`` which will show number of tests in the report.
+
+.. dropdown:: ``buildtest report --row-count``
+
+    .. command-output:: buildtest report --row-count
+
+You may find it useful to fetch all failed records and determine pass/fail criteria as part of your shell script, this can be done by running ``buildtest report --row-count --fail`` as shown below.
+The ``--row-count`` option also works with the ``--filter`` option.
```

Comment:
Can you move this statement `The --row-count ...` after the output since you mention above `as shown below` so user should see output